### PR TITLE
[PLAT-2038] Improved experimental scene creation

### PR DESCRIPTION
**Pull request by brentav**
## Summary
This change set introduces an improved experimental approach for scene item creation when using `create-scene` with the `--experimental` flag. Scene items will be created in stages based on depth. Run with the `-v` option for more detailed status output.

## Test Plan
Test `create-scene` with and without the `--experimental` and `-v` flags.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ $ npm install -g @vertexvis/cli
 $ vertex COMMAND
 running command...
 $ vertex (-v|--version|version)
-@vertexvis/cli/0.16.2 darwin-x64 node-v16.16.0
+@vertexvis/cli/0.16.3 darwin-x64 node-v16.7.0
```
Comment (MadisonEhlers-Vertex) [inline README.md:42]: Why the node downgrade?

File: src/commands/create-scene.ts [hunk 2]
```diff
@@ -255,16 +293,78 @@ f79d4760-0b71-44e4-ad0b-22743fdd4ca3
...
+            Record<string, SceneItem>
+          >((itemMap, item) => {
+            itemMap[item.suppliedId] = item;
+            return itemMap;
+          }, {});
+          let cursor: string | undefined;
+          let itemsRemain = items.length > 0;
+          while (itemsRemain) {
+            // eslint-disable-next-line no-await-in-loop
+            const res = await getPage(() =>
```
Comment (MadisonEhlers-Vertex) [inline src/commands/create-scene.ts:311]: What happens here if there is an item that gets created on the first page, but we've already passed the first page in the pagination process?

Comment (MadisonEhlers-Vertex) [reply src/commands/create-scene.ts:311]: Discussed on Slack - recommending here to split this while loop into its own function that fetches all the items for readability.

**Review by MadisonEhlers-Vertex — APPROVED** [review]

File: src/commands/create-scene.ts (hunk 2 above)
Comment (brentav) [reply src/commands/create-scene.ts]: Addressed.

**Pull request merged**